WordPress/src/main/java/org/wordpress/android/ui/ShareIntentReceiverFragment.java

```diff
@@ -185,7 +187,7 @@ public HeaderViewHolder onCreateViewHolder(LayoutInflater layoutInflater, ViewGr
             }
 
             @Override
-            public void onBindViewHolder(HeaderViewHolder holder, SiteList sites) {
+            public void onBindViewHolder(HeaderViewHolder holder, List<SiteRecord> sites) {
```

Comment:
I've also tested the share functionality and the `ShareIntentReceiverFragment` works as expected 🎉